src/Uno.UWP/Networking/Connectivity/Internal/NetworkReachability.iOSmacOS.cs

```diff
@@ -166,13 +173,13 @@ internal ReachabilityListener()
 
         internal void Dispose()
         {
-            defaultRouteReachability?.Dispose();
-            defaultRouteReachability = null;
-            remoteHostReachability?.Dispose();
-            remoteHostReachability = null;
-
+            _defaultRouteReachability?.Dispose();
+            _remoteHostReachability?.Dispose();
+			_networkMonitor.SnapshotHandler = null;
+			_networkMonitor?.Dispose();			
```

Comment:
you handle a null `_networkMonitor` here but not on the previous line
IOW if `null` then the previous line will throw a `NullReferenceException`

Reply:
Yes yes, this will still change, the iOS part is still in flux 😀